packages/rum-core/src/domain/configuration.spec.ts

```diff
@@ -91,6 +96,66 @@ describe('validateAndBuildRumConfiguration', () => {
     })
   })
 
+  describe('trackFrustrations', () => {
+    describe('without frustration-signals flag', () => {
+      it('defaults to false', () => {
+        expect(validateAndBuildRumConfiguration(DEFAULT_INIT_CONFIGURATION)!.trackFrustrations).toBeFalse()
+      })
+
+      it('the initialization parameter is ignored, no matter its value', () => {
+        expect(
+          validateAndBuildRumConfiguration({ ...DEFAULT_INIT_CONFIGURATION, trackFrustrations: true })!
+            .trackFrustrations
+        ).toBeFalse()
+        expect(
+          validateAndBuildRumConfiguration({ ...DEFAULT_INIT_CONFIGURATION, trackFrustrations: 'foo' as any })!
+            .trackFrustrations
+        ).toBeFalse()
+      })
+
+      it('does not impact "trackInteractions"', () => {
+        expect(
+          validateAndBuildRumConfiguration({ ...DEFAULT_INIT_CONFIGURATION, trackFrustrations: true })!
+            .trackInteractions
+        ).toBeFalse()
+      })
+    })
+
+    describe('with frustration-signals flag', () => {
+      beforeEach(() => {
+        updateExperimentalFeatures(['frustration-signals'])
+      })
+      afterEach(() => {
+        resetExperimentalFeatures()
+      })
+
+      it('defaults to false', () => {
+        expect(validateAndBuildRumConfiguration(DEFAULT_INIT_CONFIGURATION)!.trackFrustrations).toBeFalse()
+      })
+
+      it('the initialization parameter is set to provided value', () => {
+        expect(
+          validateAndBuildRumConfiguration({ ...DEFAULT_INIT_CONFIGURATION, trackFrustrations: true })!
+            .trackFrustrations
+        ).toBeTrue()
+      })
+
+      it('the initialization parameter the provided value is cast to boolean', () => {
+        expect(
+          validateAndBuildRumConfiguration({ ...DEFAULT_INIT_CONFIGURATION, trackFrustrations: 'foo' as any })!
+            .trackFrustrations
+        ).toBeTrue()
+      })
+
+      it('implies "trackInteractions"', () => {
+        expect(
+          validateAndBuildRumConfiguration({ ...DEFAULT_INIT_CONFIGURATION, trackFrustrations: true })!
+            .trackInteractions
+        ).toBeTrue()
+      })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**❓ question:** ‏ this exact expectation is already tested by `the initialization parameter is set to provided value`‏ test, any reason to have it twice?
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->

Reply:
It's not quite the same, we are testing the `trackInteractions` value here

Reply:
🤦